pkg/cmd/roachtest/test_runner.go

```diff
@@ -334,9 +334,9 @@ func (r *testRunner) Run(
 			//  TODO(bhaskar): remove this once we have more usage details
 			//  and more convinced about using spot VMs for all the runs.
 			if roachtestflags.Cloud == spec.GCE &&
-				tests[i].Benchmark &&
```

Comment:
Is the deliberately removed? I still see [this](https://github.com/nameisbhaskar/cockroach/blob/master/pkg/cmd/roachtest/test_runner.go#L347-L348) on the master branch and some differences.

Reply:
This is done deliberately as we do not have the AWS changes.